packages/trpc/package.json

```diff
@@ -17,10 +17,13 @@
     "@trpc/next": "^10.36.0",
     "@trpc/react-query": "^10.36.0",
     "@trpc/server": "^10.36.0",
+    "bad-words": "^3.0.4",
```

Comment:
We had to do something similar at iCodeThis, but instead of URLs, we had to make sure the user comments didn't contain profanity.

We created an array with profane words, rather than installing a package. So, that made me think - do we need to install a library for this? Also, the words list seems really small, unless I'm missing something.

Just thinking out loud. 😃

Reply:
I just think it's weird to have profanity directly in our repository where it may show up in searches

Reply:
Gotcha